src/main/java/uk/gov/hmcts/reform/civil/helpers/judgmentsonline/JudgmentOnlineMapper.java

```diff
@@ -57,8 +57,8 @@ private Integer getNextJudgmentId(CaseData caseData) {
     }
 
     private boolean isHistoricJudgment(JudgmentDetails activeJudgment) {
-        return JudgmentState.CANCELLED.equals(activeJudgment.getState())
-            || JudgmentState.SET_ASIDE_ERROR.equals(activeJudgment.getState())
-            || JudgmentState.SET_ASIDE.equals(activeJudgment.getState()) ? true : false;
+        return uk.gov.hmcts.reform.civil.model.judgmentonline.JudgmentState.CANCELLED.equals(activeJudgment.getState())
+            || uk.gov.hmcts.reform.civil.model.judgmentonline.JudgmentState.SET_ASIDE_ERROR.equals(activeJudgment.getState())
+            || uk.gov.hmcts.reform.civil.model.judgmentonline.JudgmentState.SET_ASIDE.equals(activeJudgment.getState());
```

Comment:
i feel like it was cleaner before, can we not just import JudgmentState?

Reply:
yeah will update this to import JudgmentState